legal-api/scripts/manual_db_scripts/legal_name_change/legal_name_updates.sql

```diff
@@ -403,12 +403,17 @@ select distinct ph.id
                     and ph.party_type = 'organization'
                     and
                  le.id is not null)                                                             is_business_party_org_match,
-                (r.id is null
+                ((r.id is null
                     and pr.filing_id is null
                     and le.id is null
                     and ph.party_type = 'organization'
                     and (ph.identifier is not null and ph.identifier != '' and ph.identifier not like 'FM%')
-                    )                                                                           is_business_party_colin_entity,
+                    ) or (r.id is null
+                      and pr.filing_id is null
+                      and ph.party_type = 'organization'
+                      and le.id is null
+                      and (ph.identifier is null or ph.identifier = '' or ph.identifier like 'FM%')
+                    ))                                                                          is_business_party_colin_entity,
```

Comment:
This column `is_business_party_colin_entity` is what dictates the colin entities that'll be created and be linked to.

The idea was to add the condition where when there's no identifier (either '' or null), to drop it in the colin entities table. This is basically what the `is_business_party_org_no_match` dictates. That column though isn't being used anywhere from what I checked.

Reply:
After thinking about this some more, i think we might be able to simplify this to the following sql.  Anything that doesn't match against LEAR should just go into `colin_entities`.

```sql
(r.id is null
                    and pr.filing_id is null
                    and le.id is null
                    and ph.party_type = 'organization')                                                                           is_business_party_colin_entity
```

Can you update to use this in the two relevant locations in this script and re-run the script to see what numbers we get?

Reply:
Argus, I've made the update you mentioned and you're absolutely right. I'm getting the same results.
For legal entities table:
![same results legal entities](https://github.com/bcgov/lear/assets/122301442/a918b85b-f277-4f3b-a67d-b9916583dee4)
For colin entities table:
![same results colin entities](https://github.com/bcgov/lear/assets/122301442/982b8925-73e6-4474-ad84-254537c78b94)
For entity roles:
![entity roles same results](https://github.com/bcgov/lear/assets/122301442/c114f012-9147-4da9-8e11-43e4dfd5d5ed)

Hence, they're pretty much identical to before.

Reply:
great, thanks for checking!